pkg/pod/container.go

```diff
@@ -235,6 +235,31 @@ func (builder *ContainerBuilder) WithEnvVar(name, value string) *ContainerBuilde
 	return builder
 }
 
+// WithVolumeMount  adds a pod volume mount inside the container.
```

Comment:
Remove extra space between `WithVolumeMount` and `adds`

Reply:
removed, thanks